clients/py/v3io_frames/client.py

```diff
@@ -235,6 +235,8 @@ def read(self, backend='', table='', query='', columns=None, filter='',
             True - return a DataFrames iterator;
             False (default except for get_raw=True) - return a single DataFrame
         get_raw (Optional) : bool
+            True to return the data in raw format instead as pandas DataFrames
```

Comment:
no need for this line, it's mentioned 2 lines down

Reply:
@dinal This line was actually intended to replace the existing detailed description, but I see now that I forgot to delete that description. => Done now in commit af138f7.